### PR TITLE
SecurityAlert Entity Mapping Updates

### PR DESCRIPTION
**Pull request by ddamenova**
Change(s):
   - Manny and Diana updating account updating by splitting UPNSuffix and account name
- Changing authors of entity mappigns
- Removing extraneous custom entity named columns

   Reason for Change(s):
   -  Adjusting entity mappings for correlations

   Version Updated:
   - yes

   Testing Completed:
   - yes

   Checked that the validations are passing and have addressed any issues that are present:
   - no

### CODE REVIEW

**Comment by v-rbajaj** [comment]
Hi @ddamenova, thanks for raising this PR, we will review this PR by 20 Nov 2023.

File: Detections/SecurityAlert/Massdownload_USBFileCopy.yaml [hunk 1]
```diff
@@ -83,6 +83,8 @@ query: |
   | where DeviceFileEvent_Files startswith USB_DriveLetter
   | summarize USB_Drive_MatchedFiles = make_set_if(DeviceFileEvent_Files, DeviceFileEvent_Files startswith USB_DriveLetter) by Alert_TimeGenerated, USB_TimeGenerated, UserId, AADUserId, DeviceId, DeviceName, IPAddress, CloudAppEvents_Details = tostring(CloudAppEvents_Details), DeviceFileEvents_Details = tostring(DeviceFileEvents_Details), USB_Details = tostring(USB_Details)
   | extend InitiatingProcessFileName = tostring(split(todynamic(DeviceFileEvents_Details).InitiatingProcessFolderPath, "\\")[-1]), InitiatingProcessFolderPath = tostring(todynamic(DeviceFileEvents_Details).InitiatingProcessFolderPath)
+  | extend HostName = tostring(split(DeviceName, ".")[0]), DomainIndex = toint(indexof(DeviceName, '.'))
+  | extend HostNameDomain = iff(DeviceName != -1, substring(DeviceName, DomainIndex + 1), DeviceName)
 entityMappings:
   - entityType: Account
     fieldMappings:
```
Comment (shainw) [inline Detections/SecurityAlert/Massdownload_USBFileCopy.yaml:90]: FullName needs to be fixed here, should be broken into Name and UPNSuffix

File: Detections/SecurityEvent/AccessibilityFeaturesModification.yaml [hunk 2]
```diff
@@ -26,22 +26,26 @@ query: |
...
+        columnName: HostName
+      - identifier: NTDomain
+        columnName: HostNameDomain
+version: 1.0.2
 kind: Scheduled
 metadata:
     source:
         kind: Community
     author:
-        name: Vasileios Paschalidis
```
Comment (shainw) [inline Detections/SecurityEvent/AccessibilityFeaturesModification.yaml:44]: Don't make this change, this is not a MSecR person and we don't want to remove credit for external contributors.

File: Detections/SecurityEvent/AdminSDHolder_Modifications.yaml [hunk 3]
```diff
@@ -23,22 +23,29 @@ query: |
...
+      - identifier: NTDomain
+        columnName: HostNameDomain
+version: 1.0.3
 kind: Scheduled
 metadata:
     source:
         kind: Community
     author:
-        name: Vasileios Paschalidis
+        name: Microsoft Security Research
```
Comment (shainw) [inline Detections/SecurityEvent/AdminSDHolder_Modifications.yaml:48]: Same here, don't make this change.  Only do this for MSecR folks.

File: Detections/SecurityEvent/GroupCreatedAddedToPrivlegeGroup_1h.yaml [hunk 4]
```diff
@@ -92,6 +92,8 @@ query: |
   | join (
   GroupAddition
   ) on GroupSid
+  | extend HostName = tostring(split(GroupCreateComputer, ".")[0]), DomainIndex = toint(indexof(GroupCreateComputer, '.'))
+  | extend HostNameDomain = iff(DomainIndex != -1, substring(GroupCreateComputer, DomainIndex + 1), GroupCreateComputer)
```
Comment (shainw) [inline Detections/SecurityEvent/GroupCreatedAddedToPrivlegeGroup_1h.yaml:96]: Need to extend GroupCreateSubjectAccount for Account: Name and UPNSuffix to replace FullName

**Review by shainw — CHANGES_REQUESTED** [review]
A few things to fix up, but looking good.

**Review by shainw — APPROVED** [review]

**Pull request merged**